Automattic/wp-calypso
# Make Business plan popular in "Business plans" tab

#### Changes proposed in this Pull Request

* Make Business plan popular in "Online stores..." tab

<img width="968" alt="zrzut ekranu 2018-11-19 o 13 55 54" src="https://user-images.githubusercontent.com/205419/48708434-d99f8500-ec02-11e8-80be-c05a4783fe61.png">

#### Testing instructions

* Go to /plans and plans step during the signup as user of free, premium, and business plan and confirm that Premium plan is marked as popular in "personal" plans tab, and that Business plan is marked as popular in "Online stores.." plans tab
